src/cpu.rs

```diff
@@ -60,6 +60,10 @@ pub(crate) fn features() -> Features {
     Features(())
 }
 
+#[cfg_attr(
+    not(any(target_arch = "aarch64", target_arch = "arm")),
+    allow(dead_code)
+)]
```

Comment:
I'm not sure why this would be needed in this PR, but not needed already. Maybe this can be removed now?